packages/visx-demo/src/sandboxes/visx-xychart/ExampleControls.tsx

```diff
@@ -742,41 +746,43 @@ export default function ExampleControls({ children }: ControlsProps) {
           </label>
         </div>
         {/** animation trajectory */}
-        <div>
-          <strong>axis + grid animation</strong>
-          <label>
-            <input
-              type="radio"
-              onChange={() => setAnimationTrajectory('center')}
-              checked={animationTrajectory === 'center'}
-            />
-            from center
-          </label>
-          <label>
-            <input
-              type="radio"
-              onChange={() => setAnimationTrajectory('outside')}
-              checked={animationTrajectory === 'outside'}
-            />
-            from outside
-          </label>
-          <label>
-            <input
-              type="radio"
-              onChange={() => setAnimationTrajectory('min')}
-              checked={animationTrajectory === 'min'}
-            />
-            from min
-          </label>
-          <label>
-            <input
-              type="radio"
-              onChange={() => setAnimationTrajectory('max')}
-              checked={animationTrajectory === 'max'}
-            />
-            from max
-          </label>
-        </div>
+        {!prefersReducedMotion && (
```

Comment:
I think it might be nicer to always render these controls, and then just disable them and add a message explaining why so that people who visit this page with prefers reduced motion are still able to see how it can be used. It would be even better if there was a way for these people to override and enable the animations anyway if they want to see the demo with the animations without having to mess with their OS settings or whatever.

Reply:
good call, I updated the examples to have an `enable animation` toggle with a default determined by `prefersReducedMotion`.

![image](https://user-images.githubusercontent.com/4496521/106060948-8e6a4200-60a9-11eb-8c9b-bc935aa3255c.png)